package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.47.2",
+  "version": "2.47.3",
```

Comment:
I think it should be 2.48.0, since we're actually breaking MessagingInput by removing a prop.

Reply:
Oh okay, in that case its a major change right, not minor?

Reply:
The component isn't in use in production, so it isn't breaking anything, come to think of it. Let's just leave it as v2.47.3, then. 🤷‍♀️